basic-integration/surfly-integration-with-tag-manager.md

```diff
@@ -0,0 +1,177 @@
+<a href="https://www.surfly.com/">![logo](../images/logosmall.png)</a>
+#Surfly integration with Tag Manager
+
+Tag management systems are mostly used to inject Javascript into websites for marketing purposes. But with most tag managers it is possible to inject any type of html into your website, and not only scripts. You can add text, buttons, etc. and even implement Surfly through a tag management system.
+
+In this chapter we'll show you how to add a basic Surfly implementation to your website using Google Tag Manager.
+
+####The Features
+
+Within this example we make use of the [session-id approach](../tutorial/advanced-integration.md), which gives you the following use-case:
+
+- The user is already chatting with an agent, and is directed to the "Get Help"-button to start a Surfly co-browsing session
+
+![get help](../images/get-help-button.png)
+
+- After the user clicks the "Get-Help"-button, a Surfly session is initiated, and the button now shows the session-id (which is by default a four-number code)
+- The user can now pass the session-id to the agent. The agent then uses the session-id to identify the correct session from the queue in his/her admin panel
+
+![chat box](../images/olark-chat-box.png)
+
+- The agent joins the session, and the session now becomes visible to both users
+
+####How it works
+
+With Google Tag Manager you are able to inject HTML into your website. As long as you put it in between the correct tags, you can add divs, css, javascript, etc., to your website's page(s). The tag you inject through GTM can even contain more than one script, and even scripts of a different nature.
+
+In short, you add a tag to your website as follows:
```

Comment:
Yup I get what you mean. I guess it could be useful from a marketing standpoint. Venesha suggested it but I'll check with Nicholas as well